roamance/src/components/social/post/post-card.tsx

```
@@ -0,0 +1,787 @@
+                      <span className="relative flex h-8 w-8 items-center justify-center shrink-0">
+                        <span className="absolute inset-0 rounded-full bg-gradient-to-r from-violet-500 to-purple-500 opacity-85"></span>
+                        <Sparkles className="h-3.5 w-3.5 text-white z-10" />
+                        <motion.span
+                          className="absolute inset-0 rounded-full bg-gradient-to-r from-purple-500 to-violet-600"
+                          animate={{
+                            scale: [1, 1.2, 1],
+                            opacity: [0.7, 0.9, 0.7],
+                          }}
+                          transition={{
+                            duration: 2,
+                            repeat: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)